test-http/src/test/cve_id.py

```diff
@@ -12,8 +12,7 @@
 CVE_ID_URL = '/api/cve-id'
 cve_id = 'CVE-1999-0001'
 
-
-def test_get_cve_id():
+def test_get_cve_id_id():
```

Comment:
```suggestion
def test_get_cve_id():
```

Reply:
I believe duplicate test names mean only one of those tests will be run. In general, you don't want to ever overload a function name if you can help it.